oidc-ui/public/locales/en.json

```diff
@@ -207,9 +207,9 @@
     "login_heading": "Login using eSignet",
     "verify_heading": "Verify using eSignet",
     "link_heading": "Link using eSignet",
-    "login_subheading": "Please use your credentials to login",
-    "verify_subheading": "Please provide your credentials for verification",
-    "link_subheading": "Please provide your credentials to link with your account",
+    "login_subheading": "<strong>{{clientName}}</strong> is requesting authentication for login",
+    "verify_subheading": "<strong>{{clientName}}</strong> is requesting authentication for verification",
+    "link_subheading": "<strong>{{clientName}}</strong> is requesting authentication for link your account",
```

Comment:
<strong>{{clientName}}</strong> is requesting authentication **to** link your account
change "for" -> "to"